framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2530,4 +2540,28 @@ private void printGitProperties() {
             System.out.println("IOException while reading git.properties: " + e.getMessage());
         }
     }
+
+    /** Extract the version of the Checker Framework */
+    private String getCheckerVersion() {
+        String version = null;
+        Properties p = new Properties();
+        String RLS_FILE = "/docs/developer/release/release.properties";
+        p.putAll(getProperties(getClass(), RLS_FILE));
```

Comment:
Rather than creating an empty Properties on line 2547 and then merging a Properties into it, can this line be
```
Properties p = getProperties(getClass(), RLS_FILE);
```
?